client/src/components/Input.js

```diff
@@ -31,6 +32,7 @@ const ButtonIcon = styled.button`
   height: 20px;
   width: 20px;
   border: none;
+  padding: 0px;
```

Comment:
You can use "0" instead of "0px" - just a small thing I noticed :D 
Maybe you should move basic styling to the globalstyles, so you don't crowd the styledcomponents too much

Reply:
You are right. THX 👍🏻